Wikipedia/Code/UIApplication+SystemSettings.swift

```diff
@@ -8,4 +8,12 @@ extension UIApplication {
         }
         self.open(settingsURL as URL, options: [:], completionHandler: nil)
     }
+
+    @objc func wmf_openGeneralSystemSettings() {
```

Comment:
A couple of things here:

1. I notice that even when tapping the X button on the "No internet connection" banner, this still kicks the user out to Settings. This might need some adjusting so that only the "Go to Settings" button navigates them.
2. It also feels a bit odd that this goes to the Wikipedia-specific page in Settings when the error message says "No internet connection". From some light googling, we don't seem to have the ability to go to the root Settings page, where they could actually look into their connection. So before fixing the previous issue, mention this limitation to design in case they want to scrap the Settings button entirely.

Reply:
I'll check with design at today's meeting!

Reply:
Removed the button, method and string